### PR TITLE
Fixing imports

### PR DESCRIPTION
**Pull request by jmansdorfer**
Various fixes to docs.

### CODE REVIEW

File: pages/index.mdx [hunk 1]
```diff
@@ -54,6 +54,8 @@ import { Tab, Tabs } from 'nextra-theme-docs'
   <Tab>
     ```python filename="main.py" copy
     import os
+	import json
```
Comment (dwhitena) [inline pages/index.mdx:57]: Looks like there might be some indentation issues here.

File: pages/reference/factuality.mdx [hunk 2]
```diff
@@ -15,6 +15,8 @@ import { Tab, Tabs } from 'nextra-theme-docs'
   <Tab>
     ```python filename="main.py" copy
     import os
+	import json
```
Comment (dwhitena) [inline pages/reference/factuality.mdx:18]: indentation

File: pages/reference/toxicity.mdx [hunk 3]
```diff
@@ -10,6 +10,8 @@ import { Tab, Tabs } from 'nextra-theme-docs'
   <Tab>
     ```python filename="main.py" copy
     import os
+	import json
```
Comment (dwhitena) [inline pages/reference/toxicity.mdx:13]: indentation.

**Review by dwhitena — CHANGES_REQUESTED** [review]

File: pages/index.mdx (hunk 1 above)
Comment (jmansdorfer) [reply pages/index.mdx]: Looks like a weird error in github. I fixed it in the editor here.

**Pull request merged**